SimPPS/RPDigiProducer/plugins/BuildFile.xml

```diff
@@ -0,0 +1,7 @@
+<use   name="SimTracker/Common"/>
+<use   name="RecoCTPPS/PixelLocal"/>
```

Comment:
@mundim , sorry I miss this before, dependence on RecoCTPPS is not needed.

Reply:
I'm the one to be sorry. I have done a detailed analysis on the buildfile and this scaped my attention. it was needed to be replaced by other more "primitive" packages, like CondFormats, DataFormats, etc.